package.json

```diff
@@ -6,27 +6,26 @@
     "@microsoft/azure-iot-ux-fluent-controls": "^6.0.3",
     "@microsoft/azure-iot-ux-fluent-css": "6.0.0",
     "classnames": "^2.2.6",
-    "i18next": "^12.0.0",
+    "i18next": "^14.0.1",
     "i18next-browser-languagedetector": "^2.2.4",
     "i18next-xhr-backend": "^1.5.1",
-    "react": "^16.7.0",
+    "react": "^16.8.1",
     "react-app-polyfill": "^0.2.0",
-    "react-dom": "^16.7.0",
-    "react-i18next": "^8.3.8",
-    "react-router-dom": "^4.3.1"
+    "react-dom": "^16.8.1",
+    "react-i18next": "^10.0.0",
+    "react-router-dom": "^4.4.0-beta.6"
```

Comment:
can you add a comment to explain what we need in beta.6 that's not part of 4.4.0

Reply:
Reverted back to 4.3.1. 4.4 is supposed to be a maintenance release that fixes some react dev warnings with prop types and StrictMode, but since we'll get dev warnings anyway because of fluent controls using deprecated component lifecycle methods, we might as well use 4.3 till they publish the stable version of 4.4.